composer.json

```diff
@@ -63,6 +64,7 @@
     },
     "scripts": {
         "pint": "pint",
+        "rector": "rector process --dry-run",
```

Comment:
ℹ️ Rector is a bit overzealous around some refactoring at the moment (e.g. making some classes read-only even though they extend non-read-only classes), so keeping `--dry-run` for now